integrations/db/pgsql/src/main/resources/META-INF/services/io.helidon.common.mapper.spi.MapperProvider

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
There should be no service files in Helidon. Only declare the service in `module-info.java`